Common/cpp/SharedItems/Shareable.cpp

```diff
@@ -101,6 +154,20 @@ jsi::Object ShareableValue::createHost(jsi::Runtime &rt, std::shared_ptr<jsi::Ho
   return jsi::Object::createFromHostObject(rt, host);
 }
 
+jsi::Value createFrozenWrapper(ShareableValue *sv, jsi::Runtime &rt, std::shared_ptr<FrozenObject> frozenObject) {
+  jsi::Object __reanimatedSharedRef = sv->createHost(rt, frozenObject);
```

Comment:
I think that with this approach we no longer need host objects to actually expose any properties. We only need host objects to keep references to data that we need to use when running `toJS` for the first time.

Reply:
I don't think so. We still have to control access to mutable and remote.